modules/openapi-generator/src/main/resources/swift5/Podspec.mustache

```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   {{#podDocumentationURL}}
   s.documentation_url = '{{podDocumentationURL}}'
   {{/podDocumentationURL}}
-  s.source_files = '{{projectName}}/Classes/**/*.swift'
```

Comment:
@aymanbagabas another place that I remembered that I think we need to update the path is in the `XcodeGen.mustache` file, the `sources` property